pcdsdevices/gauge.py

```diff
@@ -0,0 +1,64 @@
+"""
+Standard classes for LCLS Gauges
+"""
+import logging
+from ophyd import EpicsSignal, EpicsSignalRO, Device, Component as Cpt
+
+logger = logging.getLogger(__name__)
+
+class BaseGauge(Device):
+    """
+    Vacuum gauge
+
+    A base class for a device with two limits switches controlled via an
+    external command PV. This fully encompasses the controls `Stopper`
+    installations as well as un-interlocked `GateValves`
+
+    Parameters
+    ----------
+    prefix : ``str``
+        Full Gauge base PV
+
+    name : ``str``
+        Alias for the gauge
+    """
+    # 
+    pressure = Cpt(EpicsSignalRO, ':PMON', kind='normal')
+    status = Cpt(EpicsSignalRO, ':STATUSMON', kind='normal')
+    pressure_status = Cpt(EpicsSignalRO, ':PSTATMON', kind='normal')
+    pressure_status_enable = Cpt(EpicsSignal, ':PSTATMSP', kind='normal')
+
+    def __repr__(self):
+        print('Gauge is %s'%self.status)
```

Comment:
A few notes here:

1. Just `self.status` and `self.pressure` will give you the `repr` of the signals. If you want the value you have to call `get`
2. It is worth having a `try / except` pattern here. If the object is disconnected not being able to represent it can be an issue
3. `repr` should return a string not `print`